### ISSUE TITLE
Volume created with wrong reclaim policy

### ISSUE THREAD

**Issue by phertweck** [torchbox/k8s-hostpath-provisioner#2]
Thank you very much for this provisioner. It's very helpful for us.

Unfortunately I've a problem with the reclaim policy. I've created two storage classes. One using reclaim policy `delete` and one `retain`. Getting the storage classes via kubectl shows the correct classes. After adding two volume claims (one for each class) both volumes use reclaim policy `delete` although the mapping to the storage class is correct.

To sum up: Storage classes are created using the correct reclaim policy. Creating Volumes from `retain` storage class always results in volumes with `delete` reclaim policy.
